client/src/test/java/com/sinch/sdk/domains/sms/api/v1/adapters/SMSServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package com.sinch.sdk.domains.sms.api.v1.adapters;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.sinch.sdk.core.http.HttpClient;
+import com.sinch.sdk.core.models.ServerConfiguration;
+import com.sinch.sdk.models.SmsContext;
+import com.sinch.sdk.models.UnifiedCredentials;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+class SMSServiceTest {
```

Comment:
Can you also add a test for the happy path (to make sure it exists 😁) ?

Reply:
I'm too much optimistic 😄 
Added